applications/nrf5340_audio/src/modules/Kconfig

```diff
@@ -49,6 +49,53 @@ config I2S_CH_NUM
 
 endmenu # I2S
 
+#----------------------------------------------------------------------------#
+menu "Music Color Synchronization"
+
+config MUSIC_COLOR_SYNC
+	bool "Enabling RGB synchronization with music"
+	default n
+	help
+		"NRFX_PWM0" and "CMSIS_DSP_TRANSFORM" has to be enabled
```

Comment:
Probably better to add "depends on" here

Reply:
Changed from "help" to "depends on"